playbooks/roles/common/files/hosts

```diff
@@ -0,0 +1,28 @@
+127.0.0.1 localhost
```

Comment:
@Agrendalath, we may not need most of `plebia.org` entries since we only use the OpenCraft domains for almost all the production stuff.